recipes/hdf5/meta.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+  name: hdf5
+  version: 1.8.16
+
+source:
+  fn: hdf5-1.8.16.tar.gz
+  url: http://www.hdfgroup.org/ftp/HDF5/current/src/hdf5-1.8.16.tar.gz
+  md5: b8ed9a36ae142317f88b0c7ef4b9c618
+
+
+build:
+  features:
+    - vc9     # [win and py27]
+    - vc10    # [win and py34]
+    - vc14    # [win and py35]
+  detect_binary_files_with_prefix: True
+
+requirements:
+  build:
+    - python    # [win]
+    - cmake >=3.1
+    - zlib
+  run:
+    - zlib
+
+about:
+  home: http://www.hdfgroup.org/HDF5/
+  license: BSD-like (http://www.hdfgroup.org/ftp/HDF5/current/src/unpacked/COPYING)
```

Comment:
Probably should just say HDF5 Software License and leave BSD-like in parentheses.